symlinked-packages/@wordpress/notices

```diff
@@ -0,0 +1 @@
+../../gutenberg/packages/notices/src
```

Comment:
Since we're on it, let's also do the related symlink in the `symlinked-packages-in-parent` too, to be used in the web side setup, where the mobile code is a submodule to the parent Gutenberg code, cool?

Reply:
I just pushed a new commit for this and tested with "npm run test-mobile" on a separate Gutenberg repo after checking out this branch on its gutenberg-mobile submodule and tests ran OK. 👍